GALLERY-README.md

```diff
@@ -6,43 +6,122 @@ number of additional features that showcase the unique affordances of
+of object capabilities, if your program has a reference to an object,
+it can call methods on that object. If it doesn't have a reference, it
+can't. For more on object capabilities, see [here](http://habitatchronicles.com/2017/05/what-are-capabilities/). For more on ERTP,
+see a quick tutorial [here](README.md) and the ERTP interface descriptions
+[here](core/issuers.chainmail) and [here](core/assays.chainmail) and [here](core/contractHost.chainmail). 
 
-In order to sell pixels, the user must create an ask in our order book. The Gallery will always have a bid (request to buy) for all pixels, but the price should be relatively low, lower than selling to another user (this will need to be done after experimentation, not sure how to guarantee it now). 
+## A preemption hierarchy of rights
 
-Now that the user has some Dust, how can they buy their pixels? The pixel canvas will have an on-hover attribute that shows the x, y coordinates of the hovered-over pixel. The user should be able to look at the canvas to see what they want to buy, and record the coordinates. Then, they can create a bid in the order book for that pixel or sell it to the Gallery for a relatively low price. 
+All of the pixels (individual squares) on the demo webpage
+are owned by the gallery, the code that administrates the pixels. The
+gallery has the power to revoke the pixels from any user at any time. When a user
+calls `gallery!tapFaucet()`, the gallery takes the least recently
+colored pixel from its current holder (if anyone holds it) and gives
+it to the user in the form of an ERTP payment. 
 
-The user should be able to put their color rights into our ERTP covered call and other contract components and create things like options. 
+| ![The hierarchy of rights in the pixel demo](readme-assets/hierarchy-pixel-rights.png) | 
+|:--:| 
+| *The preemption hierarchy of rights in the pixel demo* |
 
-For examples of how the ERTP assets work, see the gallery tests.
+The gallery is able to revoke the pixels held by users because the
+pixel demo creates a customized version of ERTP in which rights are
+*hierarchical*. Hierarchical rights are familiar in property. For
+instance, a property owner might lease an apartment to a tenant, who
+might in turn, lease it to a subtenant. Anyone higher in the hierarchy
+can take away access from anyone lower in the hierarchy, and give it
+to someone else. 
+
+## Creating Pixels
+
+ERTP provides a `makeMint` function that takes in a configuration
+function. The pixel demo configuration implements the hierarchical
+rights and allows us to add additional methods to purses and payments.
+
+The configuration function passed to `makeMint` has a slot for an
+assayMaker function. An assay describes how amounts of a particular
+thing are added and subtracted from each other, among other things.
+The assay is used, for example, when payments are deposited in purses,
+and we must calculate what the new balance of the purse is. It should
```

Comment:
You're asking the reader to hold too deep a stack of nested concepts in their head. Don't start a general explanation of assays while in the midst of explaining how a config has a slot, where a config is used to customize a makeMint. Explain first what the pixel dust looks like from a user of its issuer, purses, and payments. Then assays and amounts. Only then, how the pixel dust uses configures a makeMint to customize it to create these user-facing semantics.

Reply:
I'm not saying you should use, specifically, the order I suggest above. It's just an example. Use your judgement.

Reply:
Given the overall level of this page, I think it is a distraction for it to explain about customizing mints at all. Just explain the pixels as an ERTP example in terms of the user-facing semantics. Explaining how one build new kinds of rights should come after being comfortable that there are many kinds of rights, and how to use them.

Reply:
Yup, agreed!

Reply:
That is removed